dace/transformation/passes/analysis.py

```diff
@@ -2,16 +2,19 @@
 
 from collections import defaultdict
 from dace.transformation import pass_pipeline as ppl
-from dace import SDFG, SDFGState
+from dace import SDFG, SDFGState, properties
 from typing import Any, Dict, Set, Tuple, Optional
 import networkx as nx
 
 
+@properties.make_properties
 class StateReachability(ppl.Pass):
     """
     Evaluates state reachability (which other states can be executed after each state).
     """
 
+    CATEGORY: str = 'Analysis'
```

Comment:
I hope you don't show these in the UI...

Reply:
No they're excluded :-)